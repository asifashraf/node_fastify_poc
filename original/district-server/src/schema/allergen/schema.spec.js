const { fetchGraphQL } = require('../../lib/test-util');

test('get allergens', async () => {
  const query = `{
    allergens {
      id
      name
    }
  }`;
  // openInGraphiql(query);
  const result = await fetchGraphQL(query);
  expect(result).toMatchSnapshot();
});
