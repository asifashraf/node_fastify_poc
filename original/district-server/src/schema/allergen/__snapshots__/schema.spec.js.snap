// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get allergens 1`] = `
Object {
  "data": Object {
    "allergens": Array [
      Object {
        "id": "cc451fc7-6585-4bdd-bcab-c32b20568924",
        "name": "Dairy",
      },
      Object {
        "id": "141afd5c-b04e-47cc-b67b-b6b57ac3702c",
        "name": "Egg",
      },
      Object {
        "id": "bd75ac10-ba7c-4cb6-85a7-0ee44f81ef2d",
        "name": "Fish",
      },
      Object {
        "id": "ef83edac-0214-4f98-b007-4eacbcf41b6d",
        "name": "Gluten",
      },
      Object {
        "id": "90768e6d-eb09-4246-a865-1c9bb0aa7654",
        "name": "Mustard",
      },
      Object {
        "id": "e4f0a7e8-dcf5-46e1-9056-e76a23d9a8d0",
        "name": "Peanuts",
      },
      Object {
        "id": "13582fa9-5d17-4a2f-9b1c-e8696c0847be",
        "name": "Sesame",
      },
      Object {
        "id": "07a37612-99e0-47d6-b0af-b7b776b25b7c",
        "name": "Shellfish",
      },
      Object {
        "id": "ba1f8ebb-7eac-49e4-8eb2-d660371aadc9",
        "name": "Soy",
      },
      Object {
        "id": "e4a561ba-897b-49a4-aee6-fa1db9e5bf0e",
        "name": "Sulfites",
      },
      Object {
        "id": "0f490bf5-4eac-413b-a5ac-445082d0a57f",
        "name": "Tree Nuts",
      },
      Object {
        "id": "ea2ae210-c363-4cd8-bbde-fabc2d17d211",
        "name": "Wheat",
      },
      Object {
        "id": "e6dc574e-7b3e-4fe0-a521-1204f10804ef",
        "name": "Latex",
      },
    ],
  },
}
`;
